app/builders/application_builder.rb

```diff
@@ -59,18 +61,21 @@ def dependent_attributes(online_application)
   end
 
   def online_applicant_attributes(online_application)
-    fields = %i[title first_name last_name date_of_birth ni_number married]
+    fields = [:title, :first_name, :last_name, :date_of_birth, :ni_number, :married]
```

Comment:
Ok. I personally prefer the old way you have some good points there. The reason why we decided to do it new way was Rubocop so not really convincing argument :) So if @garytaylor thinks that the old way ([:abc]) is better and we should keep it I happy to change Rubocop config.